metrics/Chart.yaml

```diff
@@ -32,6 +32,10 @@ dependencies:
   name: grafana
   repository: https://grafana.github.io/helm-charts
   version: 6.43.*
+- condition: dcgmeEporter.enabled
+  name: dcgm-exporter
+  repository: https://nvidia.github.io/dcgm-exporter/helm-charts
+  version: 3.1.5
```

Comment:
3.*.* support?

Reply:
dcgm latest  version is  3.1.8，   3.1.3+  is supported